tools/MakeNLogXSD/TemplateXSD.xml

```diff
@@ -174,6 +174,12 @@
       </xs:annotation>
     </xs:attribute>
 
+    <xs:attribute name="enabled" type="xs:boolean" default="true">
+      <xs:annotation>
+        <xs:documentation>Is this rule enabled?</xs:documentation>
```

Comment:
Maybe a bit too simple text. Really like input.

Reply:
Maybe "Is the rule enabled and should be included in the actual LoggingConfiguration?" (Since it will be discarded if enabled=false).

Reply:
Good one about the discard, 

After thinking, I didn't like the question form. So made it now like this: 

`Enable this rule. Note: disabled rules aren't available from the API.`